sdk/resourcemanager/Azure.ResourceManager/src/BicepModelReaderWriterOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.ResourceManager
+{
+    /// <summary>
+    /// Provides the options for reading and writing bicep.
+    /// </summary>
+    [Experimental(nameof(BicepModelReaderWriterOptions))]
```

Comment:
This is a good default name for now, but we might eventually want to unify all these with a common "Azure.Provisioning" or similar for CDK scenarios to both indicate what it's for and make it easy to enable them all simultaneously.